network_analysis/closest-facility/src/main/java/com/esri/samples/closest_facility/ClosestFacilitySample.java

```diff
@@ -86,10 +87,17 @@ public void start(Stage stage) {
 
     try {
 
-      // create a map with streets basemap and add to view
-      ArcGISMap map = new ArcGISMap(Basemap.createStreets());
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with a basemap style
```

Comment:
Same comment as above, do we want to specify the style everywhere, or is it just 'clutter'?

Reply:
We're gone with consistently adding in `create a map with the insert-name basemap style`